enabled_repos.txt

```diff
@@ -0,0 +1,9 @@
+2526
```

Comment:
I think that this file probably shouldn't be here ^_^

Reply:
Oops! ^_^ :heavy_check_mark: